lib/puppet/type/foreman_hostgroup.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+require_relative '../../puppet_x/foreman/common'
+Puppet::Type.newtype(:foreman_hostgroup) do
+  desc 'foreman_hostgroup manages hostgroups in foreman.'
+
+  instance_eval(&PuppetX::Foreman::Common::FOREMAN_COMMON_PARAMS)
+
+  def self.title_patterns
+    [
+      [
+        %r{^(.*)/(.*)$},
```

Comment:
Should this also use `.+` to disallow empty strings? Same below for the name.

Reply:
Fixed.